src/main/plugin/iso19139.ca.HNAP/inflate-metadata.xsl

```diff
@@ -206,29 +206,6 @@
 
       <xsl:apply-templates select="gmd:spatialRepresentationInfo" />
       <xsl:apply-templates select="gmd:referenceSystemInfo" />
-      <xsl:if test="not(gmd:referenceSystemInfo)">
```

Comment:
@zhngamy 
Why do you need to remove this code? It only seems to be applied if there is a reference system info and a spatial resprentation type.

Reply:
Sorry some items were cached while testing. I've removed it now. Thanks!